package-lock.json

```diff
@@ -0,0 +1,6525 @@
+{
```

Comment:
In RN community yarn is more popular package manager therefore there already is `yarn.lock` file. I think we should avoid having two lock files because it will be hard to have them in sync.